packages/astro/src/@types/astro.ts

```diff
@@ -2514,6 +2514,14 @@ export interface DevOverlayPlugin {
 	beforeTogglingOff?(canvas: ShadowRoot): boolean | Promise<boolean>;
 }
 
+export interface DevToolbarApp {
+	id: string;
+	name: string;
+	icon: Icon;
+	init?(canvas: ShadowRoot, eventTarget: EventTarget): void | Promise<void>;
+	beforeTogglingOff?(canvas: ShadowRoot): boolean | Promise<boolean>;
+}
```

Comment:
This is how you would do an alias of an existing type in TypeScript!

```suggestion
export type DevToolbarApp = DevOverlayPlugin
```

Reply:
Oh I know this, but I thought `type != interface`